style.css

```diff
@@ -26,6 +19,10 @@ header {
   height: 28px;
 }
 
+ul {
```

Comment:
- Provide a Pull Request description with the features implemented.

Reply:
Thanks, I have made the changes requested.